src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java

```diff
@@ -54,6 +56,7 @@ public void should_complete_envelope_referenced_by_queue_message() throws Except
             .pollInterval(500, TimeUnit.MILLISECONDS)
             .until(() -> hasNotificationBeenSent(zipFilename));
 
+        assertThatCode(() -> getEnvelope(zipFilename)).doesNotThrowAnyException();
```

Comment:
I had an actual blob in mind, not envelope in DB. Just to make sure it exists before we send message to `processed-envelopes` queue.

Reply:
@lgonczar Fixed